modules/update-upgrading-web.adoc

```diff
@@ -63,6 +63,36 @@ and click *Update*.
 The *UPDATE STATUS* changes to `Updating`, and you can review the progress of
 the Operator upgrades on the *Cluster Operators* tab.
 
+. If you are upgrading to this release from {product-title}
+ifdef::within[]
+4.3.0,
+endif::within[]
+ifdef::between[]
+4.2.16 or earlier,
+endif::between[]
+you must restart all Pods after the upgrade is complete. You can do this using the following command:
```

Comment:
Since this is in the web console section, I'd point out that you need `oc` to complete this step.